db/schema.rb

```diff
@@ -56,6 +56,13 @@
     t.index ["idempotency_key", "event_type"], name: "index_existing_events_on_idempotency_key_and_event_type", unique: true
   end
 
+  create_table "fees", force: :cascade do |t|
```

Comment:
Would you mind also adding some fees to the seed file? Will also be helpful for testing.

Reply:
Great idea!